CHANGES.rst

```diff
@@ -32,6 +32,16 @@ Breaking changes:
 
 New features:
 
+- Add compatibility to :rfc:`9253`:
+  
+  - Add new property types :py:class:`vUid` and :py:class:`vXmlReference`.
```

Comment:
It looks like my suggestion to add `:py` prefix was wrong. There's no links in https://icalendar--941.org.readthedocs.build/en/941/reference/changelog.html#a2-unreleased.

I think you found that we need to use this syntax for icalendar objects instead, and we need to use `:py` only for Python standard library objects?

```suggestion
  - Add new property types :class:`icalender.vUid` and :class:`icalender.vXmlReference`.
```

Reply:
> I think it would be good to have an example of each way to link to the various icalendar and standard library objects.

Yeah... Documented! :heart:

Reply:
These still don't link to the icalendar classes or methods. Unresolving.